zanata-war/src/main/webapp/WEB-INF/layout/language/contact_members_modal.xhtml

```diff
@@ -0,0 +1,125 @@
+<!--
+  ~ Copyright 2016, Red Hat, Inc. and individual contributors as indicated by the
+  ~ @author tags. See the copyright.txt file in the distribution for a full
+  ~ listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it under the
+  ~ terms of the GNU Lesser General Public License as published by the Free
+  ~ Software Foundation; either version 2.1 of the License, or (at your option)
+  ~ any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful, but WITHOUT
+  ~ ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+  ~ FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+  ~ details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public License
+  ~ along with this software; if not, write to the Free Software Foundation,
+  ~ Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA, or see the FSF
+  ~ site: http://www.fsf.org.
+  -->
+
+<div xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:h="http://java.sun.com/jsf/html"
+  xmlns:zanata="http://java.sun.com/jsf/composite/zanata"
+  xmlns:a4j="http://richfaces.org/a4j"
+  xmlns:rich="http://richfaces.org/rich"
+  xmlns:f="http://java.sun.com/jsf/core"
+  class="modal" id="contactMembersDialog" tabindex="-1" role="dialog"
+  aria-labelledby="myModalLabel" aria-hidden="true">
+
+  <script>
+
+    jQuery(document).ready(function () {
+      getContactMembersMessageField().bind('input propertychange', function() {
+        updateContactMembersSendButton();
+      });
+      getContactMembersSubjectField().bind('input propertychange', function() {
+        updateContactMembersSendButton();
+      });
+    });
+
+    function cancelContactMembers() {
+      getContactMembersMessageField().val('');
+      getContactMembersSubjectField().val('');
+      getContactMembersForm().find('#contact-members-send-button').removeClass('is-active');
+      updateContactMembersSendButton();
```

Comment:
We just went through a discussion with @broccolinisoup about a similar feature in another page (see: https://github.com/zanata/zanata-server/pull/1141/files/b0d7191536a42db2b51bc3414ea7867884608e43#r61219228)

We landed on doing the validation using regular jsf validations and foregoing jquery ones (as they can be overridden in the client). I suggest we do the same here.